addons/project/static/src/scss/project_form.scss

```diff
@@ -37,3 +37,14 @@
 .o_data_cell .o_favorite i.fa {
     font-size: 1.35em;
 }
+
+.o_form_view .o_form_sheet .oe_title .o_task_name {
+    width: 66%;
+}
+
+@media only screen and (max-width:650px) {
+    .o_form_view .o_form_sheet .oe_title .o_task_name {
+        width: 100%;
+        padding-right: 5px;
+    }
+}
```

Comment:
I don't know if the 66% is important, but you could have a similar result by adding the classes `w-100 w-md-75 pe-2` to your field in the view, meaning it would have a 75% width on desktop and 100% on mobile :)

Also don't forget the new line at the end of the file ;)